packages/internal-lib-build/configs/babel.config.js

```diff
@@ -10,7 +10,8 @@ const config = {
             require('@babel/preset-env'),
             {
                 targets: {
-                    node: 22
+                    // The _minimum_ Node version supported. See https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/upgrade-node-version.html#supported-node-versions
+                    node: '18'
```

Comment:
FYI according to the babel doc, the value should be a _string_.

Added a comment too for our future selves so we remember that the Node version in this case is the minimum version we support.